src/main/java/core/basesyntax/service/TransactionHandler.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationStrategyImpl;
+import core.basesyntax.strategy.impl.ReturnOperationStrategyImpl;
+import core.basesyntax.strategy.impl.SupplyOperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TransactionHandler {
```

Comment:
```suggestion
public class FruitTransactionHandlerImpl implements FruitTransactionHandler {
```
you should have an interface FruitTransactionHandler